ci-operator/config/openshift/external-dns-operator/openshift-external-dns-operator-main.yaml

```diff
@@ -0,0 +1,22 @@
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
```

Comment:
Any reason not to use `from_repository: true` as cluster-dns-operator does?
https://github.com/openshift/release/blob/5d78b36108622172baae3ae73a6416a12bd4b730/ci-operator/config/openshift/cluster-dns-operator/openshift-cluster-dns-operator-master.yaml#L34-L35

Reply:
Oh, this is a new feature! Looks like make new-repo doesn't take care of this yet. Let me add this. Thanks!

Reply:
Actually, we don't have a build root set in the external-dns-operator repo, so lets worry about this transition later.